test/factories/code_schools.rb

```diff
@@ -12,5 +12,15 @@
     is_partner false
     rep_name Faker::Name.name
     rep_email Faker::Internet.email
+
+    factory :code_school_with_locations do
+      transient do
+        locations_count { rand(1..15) }
+      end
+
+      after :create do |code_school, evaluator|
+        create_list(:location, evaluator.locations_count, code_school: code_school)
+      end
+    end
```

Comment:
This was handy for being able to generate a bunch of schools to fiddle with in dev. It'll probably be useful in tests, too, when the test cares about a school having a location.